frontend/src/components/StockData/StockData.jsx

```diff
@@ -0,0 +1,38 @@
+import './StockData.css'
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+
+export default function StockData({ currentStock }) {
+    const API_KEY_1 = "5DIHMIONQPZFEG6R";
+    const API_KEY_2 = "ciefl6pr01qmfas4d2hgciefl6pr01qmfas4d2i0";
+    const [stockData, setStockData] = useState({});
+    const [stockPrice, setStockPrice] = useState({});
+
+    useEffect( () => {
+        const fetchStockData = async () => {
+            try {
+                const infoUrl = `https://www.alphavantage.co/query?function=OVERVIEW&symbol=${currentStock}&apikey=${API_KEY_1}`;
+                const priceUrl = `https://finnhub.io/api/v1/quote?symbol=${currentStock}&token=${API_KEY_2}`;
+                const responseInfo = await axios.get(infoUrl);
+                const responsePrice = await axios.get(priceUrl)
```

Comment:
Let's make this two calls to not block each other  by using Promise.all.
 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

https://www.tutorialspoint.com/explain-promise-all-with-async-await-in-javascript